docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,341 @@
+
+1. If using the template, click [**Use this template** in the repository](https://github.com/dagster-io/dagster-cloud-serverless-quickstart). Finish setting up the repository on GitHub and clone it to your local environment.
+
+2. To install the dagster-cloud CLI, run:
+
+   ```shell
+   pip install dagster-cloud
+   ```
+
+3. Complete the CLI setup:
+
+   1. Complete the authentication step, including [creating a user token](/dagster-cloud/account/managing-user-agent-tokens#managing-user-tokens).
+
+   2. **If using your own Dagster code**, make sure to add `dagster-cloud` as a dependency in `setup.py`. [Click here for an example](https://github.com/dagster-io/dagster-cloud/blob/main/dagster-cloud-examples/setup.py). This is already done for you if using the Serverless quickstart template.
+
+4. Run the following to deploy your code to Dagster Cloud:
+
+   ```shell
+   dagster-cloud serverless deploy \
+         --location-name "<LOCATION_NAME>" \
+         --package-name <PACKAGE_NAME> \
+   ```
+
+When finished, [continue to the next step](#step-4-set-up-cicd).
+
+</TabItem>
+</TabGroup>
+</TabItem>
+<TabItem name="Hybrid">
+
+### Hybrid
+
+To set up Hybrid deployment and deploy your code, you'll need to:
+
+1. Set up an agent
+2. Add the code location in Dagster Cloud
+
+**Step 3.1: Create a Hybrid agent**
+
+1. The modal that displays will contain a pre-generated [agent token](/dagster-cloud/account/managing-user-agent-tokens) and info about the agents currently supported by Dagster Cloud.
+2. Follow the steps for setting up and deploying your agent:
+
+   - [Amazon Elastic Container Service (ECS)](/dagster-cloud/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc)
+   - [Docker](/dagster-cloud/deployment/agents/docker)
+   - [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+   - [Local](/dagster-cloud/deployment/agents/local)
+
+Once Dagster Cloud receives an agent heartbeat, click **Continue** to move on to the next step.
+
+**Step 3.2: Add a Dagster Cloud code location**
+
+For most Hybrid deployments - with the exception of those using a local agent - you'll need to create a Docker image containing your Dagster code and then add a code location to Dagster Cloud that references the image.
+
+1. Click **Configure a Code Location**.
+
+2. In the **Code Locations** tab that displays, click **+ Add**.
+
+3. A pre-filled YAML editor with configuration will display.
+
+4. Verify that the configuration is correct, then click **Add code location** to save.
+
+The agent will attempt to load your code. After the code is loaded, the new code location will show a **Loaded** status and a sample Dagster job will display in Dagster Cloud.
+
+When finished, [continue to the next step](#step-4-set-up-cicd).
+
+</TabItem>
+</TabGroup>
+
+---
+
+## Step 4: Set up CI/CD
```

Comment:
@slopp I'm interested in your feedback here, especially. I plan to create a more detailed guide that'll cover these steps in the near future, but wanted to put _something_ here so these changes can go out.

What do you think? How's this for the short-term?

Reply:
@erinkcochran87  I left some comments on the Vercel preview. It feels a little weird to pull out Setup CI/CD into its own step since it is ideally handled in Step 3:

- Step 3 Serverless / GitHub - CI/CD is setup
- Step 3 Serverless / Import your own project / Dagster Cloud CLI - we don't have CI/CD instructions for these folks anywhere today
- Step 3 Hybrid / could have CI/CD instructions _instead of_ the current step 3.2, which instructs them to manually setup their first code deployment. A better practice world be for them to setup CI/CD and that would lead to their first successful deployment.

Reply:
@slopp mind giving this another look? I spent some time incorporating this feedback and folding the CI/CD stuff into the code deployment step, but I think I've been looking at it too long.